docs/tools/config_system.md

```diff
@@ -222,7 +222,7 @@ It is an error for `mbed_lib.json` to override configuration parameters that wer
 
 #### Overriding target attributes
 
-Target configurations contain a set of attributes that you may manipulate with configuration. You may override these attributes as if they were a normal configuration parameter. If these attributes are cumulative, you may also manipulate them with the special `attribute_add` and `attribute_remove` attributes. Find a list of the attributes that may be overwritten in our documentation about [adding and configuring targets](/docs/development/tools/adding-and-configuring-targets.html)
+Target configurations contain a set of attributes that you may manipulate with configuration. You may override these attributes as if they were a normal configuration parameter. If these attributes are cumulative, they are a list of items that you may add to with the special attribute `attribute_add` and subtract from with `attribute_remove`. It is an error to both add and subtract the same value from a cumulative attribute. Find a list of the attributes that may be overwritten in our documentation about [adding and configuring targets](/docs/development/tools/adding-and-configuring-targets.html)
```

Comment:
What does "If these attributes are cumulative" mean? Do you mean "Attributes may be cumulative, in which case they are a list of items..."?

What is the syntax for cumulative attributes? Where do they go? (I might've forgotten a bit of context that is clear on a read-through.)

Also, missing period at the end.

Reply:
Yes. That phrasing is much better. I'll be more explicit that these attributes may be added to and subtracted from with the name of their configuration parameter + `_add` and `_subtract` respectively.